tools/preprocess-1.1.0/lib/preprocess.py

```diff
@@ -0,0 +1,847 @@
+#!/usr/bin/env python
```

Comment:
Thanks for integrating this, it's helpful! I was actually thinking about integrating a preprocessor when I first landed support for producing build/config.js.

However, to minimize the number of files in our repo, let's include just the necessary files from this third-party codebase:

* tools/preprocess-1.1.0/lib/preprocess.py
* tools/preprocess-1.1.0/LICENSE.txt
* tools/preprocess-1.1.0/README.txt